### ISSUE TITLE
NTR 'important site'

### ISSUE THREAD

**Issue by mark-jensen** [SDG-InterfaceOntology/sdgio#81]
@pdez90
> please use the definition of Key biodiversity Areas for 'important sites': 
Key Biodiversity Areas (KBA) are nationally mapped sites of global significance for biodiversity conservation that have been selected using globally standard criteria and thresholds. Their identification follows best practice protected area guidelines developed by IUCN. KBAs are a designation of Conservation International. The KBA network is unique in that it encompasses a number of other high biodiversity priority areas, including:  
- Important Bird Areas (IBA) - see below for further details on IBAs.
- Important Plant Areas (IPA)
- IUCN Important Sites for Freshwater Biodiversity
- Alliance for Zero Extinction Sites (AZE)
- Ramsar Sites

**Comment by mark-jensen** [comment]
@pbuttigieg Would this be an appropriate class for ENVO?

http://www.iucn.org/about/union/secretariat/offices/iucnmed/iucn_med_programme/species/key_biodiversity_areas/

https://en.wikipedia.org/wiki/Key_Biodiversity_Area

**Comment by pbuttigieg** [comment]
>@pbuttigieg Would this be an appropriate class for ENVO?

Yes, on it. The notion is very fuzzy though, expect this to need quite a lot of refinement and further specification, probably using classes from SDGIO (e.g. policy entities) to help define importance in this case. We can create the general class to work around though.

**Comment by pbuttigieg** [comment]
Release made [here](https://github.com/EnvironmentOntology/envo/releases/tag/v2016-02-05)
changes will take some time to percolate through OntoBee etc, but the envo.owl file is ready for import.

**Comment by pbuttigieg** [comment]
[Interesting article]( http://www.birdlife.org/worldwide/news/area-size-scotland-lost-world%E2%80%99s-most-important-sites-nature ) showing important ans protected are not as close as they should be.

**Comment by cfrancois7** [comment]
It should be interesting to develop the CRID aspect of these kind of taxonomy.
In essence, a _environmental zone_ (ENVO) is not important, but it could be classified as _important site_ through a classification process done by administration, e.g. Natura 2000. It depends of a classification, e.g. a CRID and a CRID registry from IAO ontology.

And in the case when the site is considered as "important" but is not "important" due to status change for economic issue. The site is still considered as important by people or for a community, e.g. scientific community. Maybe import the _representation_ class of the Emotional ontology (MFOEM) could be interesting.

So I proposed to import the IAO classes relative to the CRID and registry as well as the representation classes and relative object properties.
